src/features/footer/Footer.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import Responsive from 'react-responsive';
+import { getConfig } from '@edx/frontend-platform';
+import { ensureConfig } from '@edx/frontend-platform/config';
+
+// Local Components
+import DesktopFooter from './DesktopFooter';
+import MobileFooter from './MobileFooter';
+
+ensureConfig([
+  'ADA_URL',
+  'LMS_BASE_URL',
+  'LOGO_TRADEMARK_URL',
+  'PRIVACY_POLICY_URL',
+  'TERMS_OF_SERVICE_URL',
+  'COPY_RIGHT_STRING',
```

Comment:
I think we should also add SITE_NAME for completeness

and COPY_RIGHT_STRING should be COPYRIGHT_STRING